app/models/health/eligibility_inquiry.rb

```diff
@@ -0,0 +1,116 @@
+# ### HIPAA Risk Assessment
+# Risk:
+# Control:
+
+require "stupidedi"
+module Health
+  class EligibilityInquiry < HealthBase
+    before_create :assign_control_numbers
+
+    scope :pending, -> () do
+      where(result: nil)
+    end
+
+    def build_inquiry_file
+      self.inquiry ||= begin
+        build_inquiry_edi
+        convert_to_text
+      end
+    end
+
+    private def build_inquiry_edi
+      config = Stupidedi::Config.hipaa
+      b = Stupidedi::Builder::BuilderDsl.build(config)
+      hl = 0
+
+      sender = Health::Cp.sender.first
+      sender_id = "#{sender.pid}#{sender.sl}"
+      application_id = id&.to_s
+
+      company_id = 'OPENPATH00'
```

Comment:
This should probably come from either an ENV variable or a setting somewhere.  Maybe it could come from the sender record?